### DISCUSSION TITLE
Support for new Level 2 Domains issued by Vietnam

### DISCUSSION THREAD

**Discussion by nguyentt** [vercel/community#2841] [Bug Report]
### Page to Investigate

https://vercel.com/dashboard/domains

### Steps to Reproduce

Hi team,

It has been announced a couple of weeks ago that Vietnam will have level 2 domains such as .io.vn (https://hatinh.gov.vn/vi/tin-tuc-su-kien/tin-bai/16034/tu-ngay-0162023-nguoi-dan-co-the-dang-ky-su-dung-cac-ten-mien-moi-id-ai-iovn#:~:text=K%E1%BB%83%20t%E1%BB%AB%20ng%C3%A0y%2001%2F6,s%C3%A1ch%20%C6%B0u%20%C4%91%C3%A3i%20%C4%91%E1%BA%B7c%20bi%E1%BB%87t.) 

However, when entering mydomain.io.vn, Vercel still takes it as a subdomain.

Could you please update to reflect these new domains? Thank you.
Regards,
T

**Comment by smaeda-ks** [comment]
Thanks. Let me check

ref: https://github.com/publicsuffix/list/pull/1771

**Reply by smaeda-ks** [reply]
@nguyentt We will be adding the support soon but need some time. Will let you know once it's available

**Reply by smaeda-ks** [reply]
@nguyentt can you try this again?

**Comment by nguyentt** [comment]
Hey Shohei,

Thank you for getting back to me. Glad to confirm it is now working.

Best,
Trung
________________________________

**Comment by nguyentt** [comment]
Hi Shohei, 

I found Vercel is not able to Generate a SSL certificate for this domain. Perhaps some more stuffs need to be configured to make it happen. Thank you.

Trung

**Reply by smaeda-ks** [reply]
This is currently blocked by Let's Encrypt: https://community.letsencrypt.org/t/some-vn-cctld-sld-arent-recognized-and-causing-ratelimited-error/200596
